src/StoryHandler.cpp

```diff
@@ -219,13 +224,15 @@ void StoryHandler::draw(sf::RenderWindow& renderWindow)
   renderWindow.draw(_windowBorder);
   //std::cout << "drawing" << std::endl;
   _gameText->draw(renderWindow, _choiceHistory);
+  _gameText->drawComplete();
   //std::cout << "drawing end" << std::endl;
   if (_choiceActive == true)
   {
     //std::cout << "Drwaing choice..." << std::endl;
     _currentChoice->draw(renderWindow, _choiceHistory);
+    _currentChoice->drawComplete();
```

Comment:
I feel that the draw function in each of these should itself call drawComplete at the end?

Reply:
We could certainly do it that way. I was trying to set it up so that it just happened automatically after a draw function, but I'm open to doing it differently. There are certainly benefits to making it explicit at the end of the draw function (the biggest being code clarity).